docs/quickstart/classification.md

```diff
@@ -99,6 +99,27 @@ plot.confusion_matrix(y_test, y_pred_dt)
 plot.feature_importances(dtc, top_n=5, feature_names=list(dtc.feature_names_in_))
 ```
 
+Use the predicted labels or classes to plot an ROC curve.
+We support 3 main formats that allow you to quickly generate a single or a multi classification ROC curve.
+
+```
```

Comment:
let's move this to the docstring. and clarify that the first two formats are for the y_true argument and the second format is for y_score